### PR TITLE
Add scion's nixpkgs and nixos refs

### PR DESCRIPTION
**Pull request by wamirez**

### CODE REVIEW

File: projects/Scion/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      scion
+      scion-apps
+      scion-bootstrapper
+      ioq3-scion
+      pan-bindings
+      ;
+  };
+  nixos = {
+    modules.services.scion = "${sources.inputs.nixpkgs}/nixos/modules/services/networking/scion/scion.nix";
+    tests.scion = "${sources.inputs.nixpkgs}/nixos/tests/scion/freestanding-deployment/default.nix";
```
Comment (wamirez) [inline projects/Scion/default.nix:18]: I am not sure if we should link this one here, it isn't render by `overview/default.nix`. [PR](https://github.com/NixOS/nixpkgs/pull/298737/files#diff-ea2b936b45b9e9753598327dd5497b25aedd816a95c8f5593991e4dc26d03505) link with test

Comment (fricklerhandwerk) [inline projects/Scion/default.nix:18]:
```suggestion
    tests.scion = "${sources.inputs.nixpkgs}/nixos/tests/scion/freestanding-deployment/default.nix";
    examples = null;
```

There isn't any dedicated example, right? Need to skip `null` fields in the overview renderer.

**Review by fricklerhandwerk — APPROVED** [review]

**Comment by wegank** [comment]
> @wegank this requires a nixpkgs-bump, which breaks anastasis though... Could you please take a look?

Not really: https://hydra.nixos.org/job/nixos/trunk-combined/nixos.tests.scion-freestanding-deployment.x86_64-linux

File: projects/Scion/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
```
Comment (fricklerhandwerk) [inline projects/Scion/default.nix:18]:
```suggestion
    # TODO: unbreak
    # tests.scion = "${sources.inputs.nixpkgs}/nixos/tests/scion/freestanding-deployment/default.nix";
    tests = null;
```

**Pull request merged**